### PR TITLE
Property changed for Solr4/6

### PR DESCRIPTION
**Pull request by alf-wchong**
solr.solrConnectTimeout was used for Solr 1. Refer https://alfresco.slack.com/archives/CANET0B55/p1618557980268600

### CODE REVIEW

**Comment by alf-wchong** [comment]
See https://alfresco.slack.com/archives/CANLLCJSU/p1618578927088100?thread_ts=1618575810.087700&cid=CANLLCJSU
Can search team review and approve?

File: search-services/latest/config/properties.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ If you wish to change the default value of a property, add the relevant property
 | solr.solrUser=solr | The Solr user name |
 | solr.solrPassword=solr | The Solr password |
 | solr.secureComms=https | The HTTPS connection |
-| solr.solrConnectTimeout=5000 | The Solr connection timeouts in ms |
+| solr.http.connection.timeout | The Solr connection timeouts in ms |
```
Comment (aborroy) [inline search-services/latest/config/properties.md:108]: Default value is 500, we can change the line to
```
| solr.http.connection.timeout=500 | The Solr connection timeouts in ms |
```

**Review by aborroy — APPROVED** [review]
LGTM, minor improvement.

**Review by alf-wchong — COMMENTED** [review]
Yes of course.

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**